connectivity/suite.go

```diff
@@ -157,10 +157,16 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 				tests.OutsideToNodePort(),
 			)
 		ct.NewTest("pod-to-pod-encryption").
-			WithFeatureRequirements(check.RequireFeatureEnabled(check.FeatureEncryption)).
+			WithFeatureRequirements(check.RequireFeatureEnabled(check.FeatureEncryptionPod)).
 			WithScenarios(
 				tests.PodToPodEncryption(),
 			)
+		ct.NewTest("node-to-node-encryption").
+			WithFeatureRequirements(check.RequireFeatureEnabled(check.FeatureEncryptionNode),
+				check.RequireFeatureEnabled(check.FeatureEncryptionNode)).
```

Comment:
@brb Shouldn't the first constant here be `FeatureEncryptionPod` instead of `FeatureEncryptionNode`?

Reply:
Oops, good spot!

Reply:
https://github.com/cilium/cilium-cli/pull/1326